bin/capsule-certs-generate

```diff
@@ -3,53 +3,75 @@ require 'rubygems'
 require 'kafo'
 require 'tempfile'
 
-def self.temp_file(name, content)
-  f = Tempfile.open([name, '.yaml'])
-  f.write content
-  f.close
-  f
+class CapsuleCertsGenerate
+  INSTALLER_DIR = '.'
+
+  attr_accessor :answers_file, :config_file
+
+  def generate_answers_file
+    temp_file('capsule-certs-answer',
+              <<-EOF
+                certs:
+                  generate: true
+                  deploy: false
+                  group: foreman
+                  org: Default_Organization
+                capsule_certs: true
+              EOF
+             )
+  end
+
+  # rubocop: disable Metrics/MethodLength
+  def generate_config_file(ans_f)
+    temp_file('capsule-certs-config',
+              <<-EOF
+                :log_level: :debug
+                :log_dir: /var/log
+                :log_name: capsule-certs-generate.log
+                :no_prefix: true
+                :answer_file: #{ans_f.path}
+                :installer_dir: #{INSTALLER_DIR}
+                :installer_name: 'foreman-installer'
+                :module_dirs: ["./_build/modules", "#{INSTALLER_DIR}/modules", "/usr/share/foreman-installer/modules"]
+                :hook_dirs: ["#{INSTALLER_DIR}/hooks"]
+                :parser_cache_path: '/usr/share/katello-installer-base/parser_cache/capsule-certs-generate.yaml'
+                :default_values_dir: /tmp
+                :dont_save_answers: true
+                :mapping:
+                  :capsule_certs:
+                    :manifest_name: capsule
+                    :dir_name: certs
+                :order:
+                  - certs
+                  - capsule_certs
+              EOF
+             )
+  end
+
+  def initialize
```

Comment:
Nitpick - I generally think to look for this at the top since its a special function that tells you how a class is setup.